components/requests/modals/EditDoctorInformationModal.tsx

```diff
@@ -18,58 +18,75 @@ import {
   useToast,
 } from '@chakra-ui/react'; // Chakra UI
 import SuccessfulEditAlert from '@components/permit-holders/SuccessfulEditAlert'; // Successful edit alert/toast
-import { useState, useEffect, SyntheticEvent, ReactNode } from 'react'; // React
+import { useState, SyntheticEvent, ReactNode } from 'react'; // React
 import { DoctorInformationCardPhysician } from '@tools/components/internal/requests/doctor-information-card'; // Physician type
+import { useMutation } from '@apollo/client'; // Apollo Client
+import {
+  UPSERT_PHYSICIAN_MUTATION,
+  UpsertPhysicianRequest,
+  UpsertPhysicianResponse,
+} from '@tools/pages/admin/permit-holders/upsert-physician'; // Page tools
 
 type EditDoctorInformationModalProps = {
   children: ReactNode;
   readonly physician: DoctorInformationCardPhysician;
   readonly onSave: (applicationData: any) => void;
 };
 
-export default function EditDoctorInformationModal({
-  children,
-  physician,
-  onSave,
-}: EditDoctorInformationModalProps) {
+export default function EditDoctorInformationModal({ children }: EditDoctorInformationModalProps) {
   const { isOpen, onOpen, onClose } = useDisclosure();
 
   const [name, setName] = useState('');
-  const [mspNumber, setMspNumber] = useState<number | undefined>();
+  const [mspNumber, setMspNumber] = useState<number>(0);
```

Comment:
```suggestion
  const [mspNumber, setMspNumber] = useState(0);
```
Take advantage of type inference